contrib/nextcloud-sync/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "nextcloud-sync"
+pkgver = "3.14.0"
```

Comment:
it's not 3.14.0 if it's actually some tag with rc1 in the name, so this pkgver is wrong

Reply:
(yes, i know before 3.14.0_rc1 there is no qt6 support, but put that version instead of a made up one)

but even better, wait for the actual 3.14.0

Reply:
sure but it still doesn't fix the need of having a _pkgver since it is -rc1 and not _rc1

Reply:
yes it does, source= can have {pkgver.replace("_", "-")}

Reply:
(i also don't know what that has to do with the pkgver being wrong)

Reply:
> yes it does, source= can have {pkgver.replace("_", "-")}

oh yea true

Reply:
> (i also don't know what that has to do with the pkgver being wrong)

i dont get what u mean, u saying 3.14 is not a version?

Reply:
> i dont get what u mean, u saying 3.14 is not a version?

the version is `3.14.0-rc1`. this is not `3.14.0`, in the same way it is not `921840128412o1i2o4u214`

the version has to be remapped in some way that apk would accept (so, - -> _), but if you call it `3.14.0`, it gets merged as `3.14.0`, then it a) looks like an actual 3.14.0 is out (it isn't), and b) later when the actual 3.14.0 comes out you would ""upgrade"" it into the same version (which doesn't make sense)

Reply:
i see, then it is fixed, 3.14.0_rc1